x-pack/plugins/lens/public/xy_visualization/types.ts

```diff
@@ -196,6 +196,11 @@ export const yAxisConfig: ExpressionFunctionDefinition<
       types: ['string'],
       help: 'The color of the series',
     },
+    showValueLabels: {
+      types: ['boolean'],
+      default: false,
+      help: 'Whether to show value labels on the chart bars (horizontal only)',
```

Comment:
The main reason to have help text is that it appears in the Canvas editor, but we don't support Lens editing from Canvas directly, so we don't prioritize this. If we did we'd require i18n labels for all help text- and I'd prefer empty labels than ones we have to change later.